.github/codecov.yml

```diff
@@ -17,4 +17,8 @@ parsers:
 comment:
   layout: "reach,diff,flags,tree"
   behavior: default
-  require_changes: no
\ No newline at end of file
+  require_changes: no
+
+ignore:
+  - "test/**/*"     # ignore test folder
+  - "**.md"          # ignore md files
```

Comment:
nit: perhaps add an empty line to make GitHub happy

Reply:
i will add after the pipeline ends :) thanks for reviewing.